modules/dicom_archive/php/viewdetails.class.inc

```diff
@@ -51,7 +51,47 @@ class ViewDetails extends \NDB_Form
      */
     function _hasAccess(\User $user) : bool
     {
-        return $user->hasPermission('dicom_archive_view_allsites');
+        // remove the possibility to have no tarchive ID in this page
+        if (empty($_REQUEST['tarchiveID'])) {
+            // defaults to permission denied
+            return false;
+        }
+
+        // get project ID from Tarchive ID.
+        $tarchiveID = intval($_REQUEST['tarchiveID']);
+        $projectID  = self::getProjectFromTarchiveID($tarchiveID);
+        if (is_null($projectID)) {
+            return false;
```

Comment:
Should this be return true? Otherwise no one will ever be able to see it?

(Maybe a discussion for an imaging meeting?)

Reply:
That means the TarchiveID does not exist in db or is not linked to a project.. ? 
I was not sure about this. It is even possible?

Reply:
Maybe we can involve @cmadjar

Reply:
Point added to next imaging meeting.

Reply:
I'm assuming it would mean the TarchiveID is not linked to a project

Reply:
Imaging meeting: no one should see it by default. 
There should be a specific permission to see the list of "dangling TarchiveIDs" (Tarchive not assigned to any Project). Also might be good to have a front-end page for that.
It will be linked to a new issue.

Reply:
Created here: https://github.com/aces/Loris/issues/9389